libservice/src/IpAddressNetlinkChecker.cpp

```diff
@@ -115,14 +116,21 @@ bool IpAddressNetlinkChecker::isV4AddressExternal(IPv4int addr) const {
 	return true;
 }
 
-static bool ipv6AddressContainsMappedIpv4Address(const in6_addr& addr) {
+bool IpAddressNetlinkChecker::ipv6AddressContainsMappedIpv4Address(const in6_addr& addr) const {
+	for (const auto& internalRange : {"::ffff:0:0:0/96", "64:ff9b::/96"}) {
+		if (isInRange(addr, internalRange)) {
```

Comment:
ipv4 address consists of 4 bytes so likely you mean  ::ffff:0:0:0:0 (4 zeros not 3)